<launch>
  <node pkg="transfer_station" type="ros_ipc_transfer_node" name="transfer" output="screen">
    <rosparam>
      ipc_name: "transfer"
      routes:
        chatter:
          type: "position"
          direction: "ipc_to_ros"
          ros_topic: "/chatter"
          ipc_message: "chatter"
    </rosparam>
  </node>

  <!-- node pkg="transfer_station" type="ipc_talker" name="IPC_Talker" output="screen"/ -->
  <!-- node pkg="transfer_station" type="ros_listener" name="ROS_Listener" output="screen"/ -->
</launch>
